includes/class-sensei-lesson.php

```diff
@@ -600,29 +600,33 @@ public function quiz_update( $post_id ) {
 
 	} // End post_updated()
 
-	public function get_submitted_setting_value( $field = false ) {
+	/**
+	 * Get setting value from POST data.
+	 *
+	 * @access private
+	 *
+	 * @param  {string} $field Field name.
+	 *
+	 * @return string|null
+	 */
+	public function get_submitted_setting_value( $field ) {
 
 		if ( ! $field ) {
-			return;
+			return null;
 		}
 
-		$value = false;
+		$value = null;
 
-		// Since we don't do any updates here, we can ignore nonce verification.
-		if ( 'quiz_grade_type' == $field['id'] ) {
+		if ( 'quiz_grade_type' === $field['id'] && 'list' !== $_POST['post_view'] ) {
```

Comment:
Looks good, this fixes it :) But instead of excluding the list page, maybe it's safer to check for the edit page?

Reply:
Refactored here: https://github.com/Automattic/sensei/pull/3589/commits/342d7f4389331a9c4113056fbc814137ed381b53